platform-services.md

```diff
@@ -51,17 +51,17 @@ non-graphical, data-only information. Examples include:
 *   Network information
 *   Sensors
 
-You can also use the [`HostMessages`](https://docs.flutter.io/flutter/services/HostMessages-class.html)
+You can also use the [`PlatformMessages`](https://docs.flutter.io/flutter/services/PlatformMessages-class.html)
 class to invoke APIs from third-party SDKs that run on iOS and Android devices.
 
 To communicate between the
-[`HostMessages`](https://docs.flutter.io/flutter/services/HostMessages-class.html) class and
+[`PlatformMessages`](https://docs.flutter.io/flutter/services/PlatformMessages-class.html) class and
 platform-specific APIs in the host, you must add the following Flutter components into
 the host's view hierarchy.
 
-+ **On iOS:** Add a [`FlutterViewController`](https://github.com/flutter/engine/blob/master/sky/shell/platform/ios/framework/Headers/FlutterViewController.h)
++ **On iOS:** Add a [`FlutterViewController`](https://github.com/flutter/engine/blob/master/shell/platform/darwin/ios/framework/Headers/FlutterViewController.h)
```

Comment:
Good one, let's run it past @sethladd 
Right now the link checker is the way we know something on the site needs to be updated. It's being integrated into Travis CI, so all the links will be checked on every build. If no broken link, we literally won't know to go investigate and update.